File: src/minecraft/monnef/jaffas/power/block/common/BlockPower.java
/*
 * Jaffas and more!
 * author: monnef
 */

package monnef.jaffas.power.block.common;

import monnef.jaffas.food.block.BlockJaffas;
import monnef.jaffas.power.JaffasPower;
import monnef.jaffas.power.common.Reference;
import net.minecraft.block.material.Material;

public class BlockPower extends BlockJaffas {
    public BlockPower(int id, int index, Material material) {
        super(id, index, material);
        setCreativeTab(JaffasPower.CreativeTab);
    }

    @Override
    public String getModName() {
        return Reference.ModName;
    }

    @Override
    public int getDefaultSheetNumber() {
        return 5;
    }
}

File: src/minecraft/monnef/jaffas/power/block/common/BlockPower.scala
package monnef.jaffas.power.block.common

import monnef.jaffas.food.block.BlockJaffas
import monnef.jaffas.power.JaffasPower
import net.minecraft.block.material.Material
import monnef.jaffas.power.common.IconDescriptorPower

class BlockPower(_id: Int, _tex: Int, _mat: Material) extends BlockJaffas(_id, _tex, _mat) with IconDescriptorPower {
  setCreativeTab(JaffasPower.instance.creativeTab)
}